### ISSUE TITLE
Icon Design

### ISSUE THREAD

**Issue by zoombear** [ghaida/gdisjc-website#14]
Illustrator? some other tool to create our visuals?

**Comment by arc6789** [comment]
What kind of icons were these? Were they the social media icons?

**Comment by zoombear** [comment]
No, not the social media. Icons and designs for the page, itself. We discussed having icons to represent the page sections/links that a user can click and the page will scroll down to reveal the section. Something like that.

**Comment by benthinkin** [comment]
That could lead to branding in general. I wonder if GDI has any guidelines? All I’ve found is their logo.

**Comment by arc6789** [comment]
![all-icons](https://cloud.githubusercontent.com/assets/8885849/7342169/66becf92-ec64-11e4-987a-46c41b8a1208.png)

Has anyone worked on icons yet? maybe something like this?

**Comment by zoombear** [comment]
Those look so cool!!!
